### ISSUE TITLE
Some ass effects can't be removed

### ISSUE THREAD

**Issue by Ukenn2112** [GitBib/pyasstosrt#6]
### Describe the problem

Some ass effects can't be removed

### Steps to reproduce

[Ass files to be converted](https://drive.google.com/file/d/1eYxWLLM19WnyMh10QS5czkMeLb3ck5QG/view)

```python
from pyasstosrt import Subtitle

sub = Subtitle('sub.ass')
sub.export()
```

<img width="847" alt="image" src="https://user-images.githubusercontent.com/60847880/180115162-b7db274c-469c-4b40-8e1e-fddb65299cda.png">

### System configuration

* Python version: 3.6

**Comment by GitBib** [comment]
Thank you for your example. I will try to implement removal of effects. Also need to glue similar lines.

**Comment by GitBib** [comment]
We need to check. #8

**Comment by GitBib** [comment]
Released 1.3.0

**Comment by Ukenn2112** [comment]
Hi, I used the latest version and when working on another ass, I found that there are still some residual effects that have not been eliminated 😂

- [Ass files to be converted](https://drive.google.com/file/d/1rPcJgJNGNQTOnug-AgXC-whq_na5fD-O/view)

<img width="645" alt="image" src="https://user-images.githubusercontent.com/60847880/192149789-281b8489-2ce8-4c15-8ea5-8167fb833118.png">

**Comment by GitBib** [comment]
Float was forgotten in the implementation

**Comment by GitBib** [comment]
@Ukenn2112 implementation has been corrected. Need to check as I don't have ass.

**Comment by Ukenn2112** [comment]
I tried the latest commit and now it works fine

> @Ukenn2112 implementation has been corrected. Need to check as I don't have ass.

**Comment by GitBib** [comment]
Released 1.3.1
